### ISSUE TITLE
Can you share the training script about the baseline?

### ISSUE THREAD

**Issue by DRJYYDS** [hankook/CLEL#1]
This work is impressive. And even your reported results of the baseline are outstanding, so can you share the training script about the baseline? I really appreciate any help you can provide.

**Comment by sndnyang** [comment]
Hi, did you find any code to reproduce the baseline?

**Comment by DRJYYDS** [comment]
> Hi, did you find any code to reproduce the baseline?

Sorry. I havn't reproduce the result :) If you're interesting, we can discuss it.

**Comment by sndnyang** [comment]
> Sorry. I havn't reproduce the result :) If you're interesting, we can discuss it.

based on the code, I find it is easy to reproduce the baseline (FID ~ 23.5),   just remove the loss from encoder.

 [ 76000 / 100000] [Loss 0.0005] [FID 31.83 | 31.83] [NN 0.25 | 0.25]

**Comment by DRJYYDS** [comment]
> > Sorry. I havn't reproduce the result :) If you're interesting, we can discuss it.
> 
> based on the code, I find it is easy to reproduce the baseline (FID ~ 23.5), just remove the loss from encoder.
> 
> [ 76000 / 100000] [Loss 0.0005] [FID 31.83 | 31.83] [NN 0.25 | 0.25]

Thanks for your information!
